src/components/live-example/Knob/RawKnob.tsx

```diff
@@ -0,0 +1,117 @@
+import { ChangeEventHandler, useMemo, useState } from "react";
+
+import Code from "@leafygreen-ui/code";
+import { css } from "@leafygreen-ui/emotion";
+import Icon from "@leafygreen-ui/icon";
+import IconButton from "@leafygreen-ui/icon-button";
+import TextArea from "@leafygreen-ui/text-area";
+
+import { KnobProps } from "./types";
+
+const editModeWrapperStyle = css`
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: 1fr 40px;
+  justify-items: center;
+  align-items: center;
+`;
+
+const textAreaStyle = css`
+  & > textarea {
+    margin: unset;
+    padding: 8px 12px;
+  }
+`;
+
+/**
+ * Enables users to edit & input valid JSON as a knob.
+ *
+ * A user can click the `edit` button to edit it the value
+ * And then click `save` to set the value in the example
+ */
+export const RawKnob = ({
```

Comment:
When do we use Raw Knobs?

Reply:
I copied this from .design, but I don't think we need this anymore since we are omitting props.